### DISCUSSION TITLE
"Start DLQ redrive" button disabled

### DISCUSSION THREAD

**Discussion by Dashue** [getlift/lift#176] [Constructs]
Hello everyone!

I'm new to lift and have set up a fifo queue using the documentation. 
It's been running well for a week and have seen some messages get moved to the DLQ upon heavy load.

The messages are valid and I would like to mark them for processing again which I believe is done using the DLQ redrive button on the dlq queue in amazon. 

This button is disabled and my searches on google, this repository and discussions for information on how to redrive or why the button is disabled is not revealing how to proceed.

Thanks in advance

**Comment by mnapoli** [comment]
Hi, no idea about that. Lift doesn't do anything specific to enable or disable that button, I would expect that button to work like for any other queue.

However just as a reminder you can run `serverless <construct-name>:failed:retry` to redrive the failed messages (https://github.com/getlift/lift/blob/master/docs/queue.md).

**Reply by JohnLawal** [reply]
@mnapoli running `serverless <construct-name>:failed:retry` on FIFO queues currently fails with error message: `The request must contain the parameter MessageGroupId`.
I opened [this issue](https://github.com/getlift/lift/issues/189) to that effect already.
